src/main/java/usecases/War.java

```diff
@@ -0,0 +1,148 @@
+            currPlayerIndex = 1;
+            currPlayer = players[currPlayerIndex];
+            makeMove(currPlayer.getHand().removeCard());
+            currPlayerIndex = 0;
+            currPlayer = players[currPlayerIndex];
+            if (!checkWin()) {
+                if (inWar) {
+                    makeMove(currPlayer.getHand().removeCard());
+                    currPlayerIndex = 1;
+                    currPlayer = players[currPlayerIndex];
+                    makeMove(currPlayer.getHand().removeCard());
+                    currPlayerIndex = 0;
+                    currPlayer = players[currPlayerIndex];
+                }
+            }
+            Card topCard0 = (Card) playingField[0].get(playingField[0].size() - 1);
+            Card topCard1 = (Card) playingField[1].get(playingField[1].size() - 1);
+
+            this.gameOutput.sendOutput("---------------------------------------");
+            this.gameOutput.sendOutput(players[0].getName() + " flipped the card " + topCard0);
+            this.gameOutput.sendOutput(players[1].getName() + " flipped the card " + topCard1);
+
+            int topCardHierarchy0 = java.util.Arrays.asList(Hierarchy).indexOf(topCard0.getRank());
+            int topCardHierarchy1 = java.util.Arrays.asList(Hierarchy).indexOf(topCard1.getRank());
+            int winner = 2;
+
+            if (topCardHierarchy0 == topCardHierarchy1) {
+                if (inWar) {
+                    this.gameOutput.sendOutput("Both players have flipped the same rank of card, so war continues!");
+                } else {
+                    inWar = true;
+                    this.gameOutput.sendOutput("Both players have flipped the same rank of card, so war begins!");
+                }
+            } else if (topCardHierarchy0 < topCardHierarchy1) {
+                this.gameOutput.sendOutput( players[1].getName() + "'s card outranks their opponents and they win the battle!");
+                winner = 1;
+            } else {
+                this.gameOutput.sendOutput( players[0].getName() + "'s card outranks their opponents and they win the battle!");
+                winner = 0;
+            }
+
+            if (winner < 2) {
+                for (int j = 0; j < 2; j++) {
+                    for (int k = 0; k < playingField[j].size(); k++) {
+                        players[winner].addToHand((Card) playingField[j].remove(k));
+                    }
+                }
+            }
+
+            this.gameOutput.sendOutput("---------------------------------------");
+
+            gameInput.stall();
+        }
+
+        if (players[0].isHandEmpty() && players[1].isHandEmpty()) {
+            this.gameOutput.sendOutput("Somehow you two have mananged to end up in an extremely improbable draw. Congratulations!");
+        } else if (players[1].isHandEmpty()) {
+            this.gameOutput.sendOutput(players[1].getName() + " is out of cards and can no longer participate. " + players[0].getName() + " wins!");
```

Comment:
Consider pulling this out into a private method to reduce the length of this method.